web/client/.gitignore

```diff
@@ -12,3 +12,4 @@ vite.config.ts.timestamp-*
 rules.json
 cypress/screenshots/
 cypress/videos/
+web/client/static
```

Comment:
Remove this ignored folder from the list, as we do have valid content in the _web/client/static_